assets/dev/scss/frontend/_container.scss

```diff
@@ -35,13 +39,17 @@
 	height: var( --height );
 	border-radius: var( --border-radius );
 	margin: var( --margin );
-	padding: var( --padding );
+	// Custom content-width hack.
+	padding-left: calc( ( 100% - var( --content-width ) ) / 2 + var( --padding-left ) );
+	padding-right: calc( ( 100% - var( --content-width ) ) / 2 + var( --padding-right ) );
+	padding-top: var( --padding-top );
+	padding-bottom: var( --padding-bottom );
 	z-index: var( --z-index );
 	overflow: var( --overflow );
 	transition: var( --transition );
 
 	// Force center only top-level containers.
-	.elementor-section-wrap > & {
+	:is( .elementor-section-wrap, [data-elementor-type = "wp-page"] ) > & {
```

Comment:
It needs a generic solution, it can be `data-elementor-type = "wp-post"` or `data-elementor-type = "product"`, etc.

Reply:
Done